[2025-09-16T18:43:28.294+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-16T18:43:28.381+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_to_postgres_batch.wait_for_kafka scheduled__2025-09-16T18:08:15.824796+00:00 [queued]>
[2025-09-16T18:43:28.392+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_to_postgres_batch.wait_for_kafka scheduled__2025-09-16T18:08:15.824796+00:00 [queued]>
[2025-09-16T18:43:28.393+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 3
[2025-09-16T18:43:28.414+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonSensor): wait_for_kafka> on 2025-09-16 18:08:15.824796+00:00
[2025-09-16T18:43:28.437+0000] {standard_task_runner.py:57} INFO - Started process 3871 to run task
[2025-09-16T18:43:28.442+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'kafka_to_postgres_batch', 'wait_for_kafka', 'scheduled__2025-09-16T18:08:15.824796+00:00', '--job-id', '719', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmp8u7ulxwz']
[2025-09-16T18:43:28.446+0000] {standard_task_runner.py:85} INFO - Job 719: Subtask wait_for_kafka
[2025-09-16T18:43:28.472+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-16T18:43:28.526+0000] {task_command.py:410} INFO - Running <TaskInstance: kafka_to_postgres_batch.wait_for_kafka scheduled__2025-09-16T18:08:15.824796+00:00 [running]> on host d6e2a7c8b3ee
[2025-09-16T18:43:28.545+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-16T18:43:28.653+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kafka_to_postgres_batch' AIRFLOW_CTX_TASK_ID='wait_for_kafka' AIRFLOW_CTX_EXECUTION_DATE='2025-09-16T18:08:15.824796+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-16T18:08:15.824796+00:00'
[2025-09-16T18:43:28.655+0000] {python.py:72} INFO - Poking callable: <function check_kafka_message at 0x7f82c00cb5f0>
[2025-09-16T18:43:28.662+0000] {conn.py:397} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:29092 <connecting> [IPv4 ('172.23.0.4', 29092)]>: connecting to kafka:29092 [('172.23.0.4', 29092) IPv4]
[2025-09-16T18:43:28.685+0000] {conn.py:618} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:29092 <checking_api_versions_recv> [IPv4 ('172.23.0.4', 29092)]>: Broker version identified as 2.6
[2025-09-16T18:43:28.687+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:29092 <connected> [IPv4 ('172.23.0.4', 29092)]>: Connection complete.
[2025-09-16T18:43:28.693+0000] {subscription_state.py:181} INFO - Updating subscribed topics to: ('user_events',)
[2025-09-16T18:43:28.698+0000] {subscription_state.py:254} INFO - Updated partition assignment: [('user_events', 0)]
[2025-09-16T18:43:28.701+0000] {conn.py:397} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.23.0.4', 29092)]>: connecting to kafka:29092 [('172.23.0.4', 29092) IPv4]
[2025-09-16T18:43:28.702+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:29092 <connected> [IPv4 ('172.23.0.4', 29092)]>: Connection complete.
[2025-09-16T18:43:28.703+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:29092 <connected> [IPv4 ('172.23.0.4', 29092)]>: Closing connection. 
[2025-09-16T18:43:28.717+0000] {fetcher.py:418} INFO - Resetting offset for partition ('user_events', 0) to offset 0.
[2025-09-16T18:43:28.826+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:29092 <connected> [IPv4 ('172.23.0.4', 29092)]>: Closing connection. 
[2025-09-16T18:43:28.827+0000] {base.py:255} INFO - Success criteria met. Exiting.
[2025-09-16T18:43:28.845+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=kafka_to_postgres_batch, task_id=wait_for_kafka, execution_date=20250916T180815, start_date=20250916T184328, end_date=20250916T184328
[2025-09-16T18:43:28.847+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-09-16T18:43:29.137+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-09-16T18:43:29.194+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
